### PR TITLE
#1361 Add more fields to KYC process

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/shared/forms/formField/FormSelectField.tsx [hunk 1]
```diff
@@ -102,17 +102,11 @@ export class FormSelectField extends React.Component<FieldGroupProps & IOwnProps
         {extraMessage ? (
           <div className={styles.noteLabel}>{extraMessage}</div>
         ) : (
-          <div className={styles.errorLabel}>
+          <>
             {isNonValid(touched, errors, name) && (
-              <div>
-                {errors[name].includes(NONE_KEY) ? (
```
Comment (desfero) [inline app/components/shared/forms/formField/FormSelectField.tsx]: @lizonp please confirm that I can remove that code. I found that it's not used anywhere but maybe i missed something important.

File: app/lib/api/KycApi.interfaces.ts [hunk 2]
```diff
@@ -58,16 +64,16 @@ export interface IKycBusinessData {
...
 export const KycBusinessDataSchema = Yup.object().shape({
-  name: Yup.string(),
-  legalForm: Yup.string(),
-  legalFormType: Yup.string(),
-  street: Yup.string(),
-  city: Yup.string(),
-  zipCode: Yup.string(),
-  country: restrictedCountry,
-  jurisdiction: Yup.string().default("de"),
+  name: Yup.string().required("This field is required"),
```
Comment (desfero) [inline app/lib/api/KycApi.interfaces.ts]: `This field is required` will be removed during #1357

File: app/components/kyc/business/LegalRepresentative.tsx [hunk 3]
```diff
@@ -183,8 +194,10 @@ const FileUploadList: React.SFC<IProps & { lrDataValid: boolean }> = props => {
 };
 
 const BeneficialOwners: React.SFC<IProps & { lrDataValid: boolean }> = props => {
-  if (!props.lrDataValid || props.files.length === 0) return <div />;
-  if (!props.businessData || !(props.businessData.legalFormType === "corporate")) return <div />;
+  if (!props.lrDataValid || props.files.length === 0) return null;
```
Comment (Moejoe90) [inline app/components/kyc/business/LegalRepresentative.tsx]: what happens when you return null from a react component? 
It just doesn't show like an iternary statement?

File: app/components/shared/forms/formField/FormSelectCountryField.tsx [hunk 4]
```diff
@@ -189,7 +190,7 @@ const VALUES = {
   RO: "Romania",
   RU: "Russian Federation",
   RW: "Rwanda",
-  BL: "Saint BarthÉLemy",
+  BL: "Saint Barthélemy",
```
Comment (Moejoe90) [inline app/components/shared/forms/formField/FormSelectCountryField.tsx]: :+1:

File: app/components/shared/forms/formField/FormSelectCountryField.tsx [hunk 5]
```diff
@@ -1,10 +1,11 @@
 import { FieldAttributes } from "formik";
 import * as React from "react";
+import { FormattedMessage } from "react-intl-phraseapp";
 
 import { FormSelectField, NONE_KEY } from "./FormSelectField";
 
 const VALUES = {
-  [NONE_KEY]: "Please select",
+  [NONE_KEY]: <FormattedMessage id="form.select.please-select" />,
```
Comment (Moejoe90) [inline app/components/shared/forms/formField/FormSelectCountryField.tsx]: :+1:

File: app/components/kyc/business/LegalRepresentative.tsx (hunk 3 above)
Comment (desfero) [reply app/components/kyc/business/LegalRepresentative.tsx]: Right, it's a lot better to return null and render nothing than rendering empty div. At a first I thought that there is some bug in the code that makes empty div :D

File: app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts [hunk 6]
```diff
@@ -33,6 +35,8 @@ const personData: IPersonData = {
   street: "example",
   city: "example",
   country: "DE",
+  placeOfBirth: "UA",
+  nationality: "PL",
```
[the remark this answers is not in the document]
Comment (desfero) [reply app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts]: :D

**Pull request merged**